tests/run.mjs

```diff
@@ -60,16 +60,20 @@ const server = await serve(PORT);
 async function runTests() {
     let success = true;
     try {
-        success &&= await runTest("Run Single Suite", () => testEnd2End({ test: "proxy-mobx" }));
-        success &&= await runTest("Run Disabled Suite", () => testEnd2End({ tag: "disabled" }));
-        success &&= await runTest("Run Default Suite", () => testEnd2End());
+        success &&= await runEnd2EndTest("Run Single Suite", { test: "proxy-mobx" });
+        success &&= await runEnd2EndTest("Run Tag No Prefetch",  { tag: "proxy", prefetchResources:"false"});
```

Comment:
just out of interest: any particular reason to run both `proxy` items, not just mbox?

Reply:
mostly to get some additional coverage.